### ISSUE TITLE
tsmorph is trying to create directories in node modules

### ISSUE THREAD

**Issue by ArielGueta** [dsherret/ts-simple-ast#1287]
I have a simple project that manipulates some files, and saves the changes:

```ts
const project = new Project({
    tsConfigFilePath: 'tsconfig.base.json',
    compilerOptions: {
        typeRoots: [],
        types: []
    }
});

// ....

project.saveSync()
```

For some reason I get the following exception:

```
Error: EROFS: read-only file system, mkdir '/node_modules'
```

The error comes from `at TransactionalFileSystem.writeFileSync (my-path/node_modules/@ts-morph/common/dist/ts-morph-common.js:2366:14)`

I logged the `filePath` and `fileText` and that's the results:

```
/node_modules/typescript/lib/lib.es2015.core.d.ts
```

```
/// <reference no-default-lib="true"/>
interface Array<T>{find<S extends T>(predicate:(this:void,value:T,index:number,obj:T[])=>value is S,t............
```

Any idea why it tries to write to `node_modules`? Thanks!

**Comment by dsherret** [comment]
@ArielGueta what version of ts-morph are you using? I remember this used to happen in an old version, but I thought it's fixed now and I'm unable to reproduce (maybe include your tsconfig.base.json file?)

**Comment by dsherret** [comment]
I believe this is fixed as of 13.0.2: https://github.com/dsherret/ts-morph/commit/a655c659094a5cb844c8bad36cec73a5cd39f091

**Comment by ArielGueta** [comment]
I was on v11. It works now, thanks!
